tests/unit/test_mixing_matrix.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from simple_network_sim.mixing_matrix import AgeRange, MixingMatrix
+
+
+def test_AgeRange():
+    with pytest.raises(Exception) as e_info:
+        a = AgeRange("[10,20)")
+        assert a.contains(10)
+        assert a.contains(15)
+        assert not a.contains(20)
+
+        b = AgeRange(10, 20)
+        assert a == b
+        assert b.contains(10)
+        assert not b.contains(20)
+
+        c = AgeRange((10,20))
+        assert b == c
+
+        d = AgeRange("70+")
+        assert d.contains(99)
+        assert not d.contains(69)
+
+        e = AgeRange("invalid")
+    assert e_info.value.args[0] == f'Invalid age range specified: "invalid"'
+
+def test_sampleMixingMatrix(mixing_matrix):
+    mm = MixingMatrix(mixing_matrix)
+    assert mm[28][57] == 0.097096431
+    assert mm["[30,40)"][75] == 0.026352071
```

Comment:
I think this interface may be a bit confusing. If I tried to query mm["[30,35)"][75] I'd get an error, right?

Reply:
With the current implementation and reading the sample mixing matrix, it would raise a KeyError on "[30,35)" yes. I wasn't sure if there was a need to allow alternate age ranges (i.e. ones not defined in the mixing matrix) because it might imply higher accuracy data than what we actually have (i.e., someone might try running with age ranges [x, x+1) for each age x, which would "run" fine but wouldn't be using any more accurate data for the mixing matrix).

Reply:
I see, I think you're right. "[30,40)" is the age class name, shouldn't really be thought of as a range.